provision/testdata/list_flavors.stdout.txt

```diff
@@ -1,4 +1,5 @@
 INFO: Available configuration flavors:
+INFO: openshift-4.14-assisted-baremetal:	Red Hat OpenShift Container Platform 4.14 on Baremetal for Assisted Installation
```

Comment:
Please update:
Red Hat OpenShift Container Platform 4.14 on Baremetal for Assisted Installation
to:
Red Hat OpenShift Container Platform 4.14 on Baremetal Agent-based Assisted Installation [Experimental]